Caddyfile

```diff
@@ -14,7 +14,6 @@
         }
         headers Content-Type Authorization
         log_level debug
-        ttl 1000s
```

Comment:
Probably shouldn't remove this from here, since this file was meant as a "all options" example. You can make this `tls 120s` to match the default.

Also, I think the README should actually have the Caddyfile syntax instead of a link to a file, plus it should have a little explanation for each of the options to explain what they do and what kinds of options they take. Refer to the official Caddy docs to get an idea of how it should look, but basically a code block, then a bullet list for each option.

Reply:
I will revert that and write the README this today 👍

Reply:
Done ✅